new_component_todo-item.md

```diff
@@ -1,16 +1,18 @@
-# New component todo-item
+# Novo componente todo-item
 
-We will create a new component which will be used for each todo item that is displayed on the list. It will be a simple component at first, but it will grow later on. What's important is that **it will get the todo item as an input from its parent component**. This way it can be a reusable component, and not rely directly on the application's data and state.
+Nós iremos criar um novo componente  que poderá ser usado para cada 
```

Comment:
Hi @dtgfranca, this topic need adjust, maybe something like "Vamos criar um novo componente que será usado para cada item que é exibido na lista."

Reply:
Thanks, topic adjusted.